recipes/slimit/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "slimit" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/rspivak/{{ name }}/archive/{{ version }}.tar.gz"
+  sha256: "015fc1917c0eff834514fcedd4251275a3dacb10df1b49549277fe5e19de1ddf"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - ply >=3.11
+test:
+  imports:
+    - slimit
+    - slimit.tests
+
+about:
+  home: "https://github.com/rspivak/slimit"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE"
+  summary: "SlimIt is a JavaScript minifier written in Python. It compiles JavaScript into more compact code so that it downloads and runs faster."
```

Comment:
You may consider wrapping this to improve readability.

Reply:
I'll fix it in the merged repo. Thanks for catching that